src/helpers/text_helper.ts

```diff
@@ -224,6 +228,90 @@ export function getContextFontSize(font: string): Pixel {
   return Number(font.match(pxRegex)?.[1]);
 }
 
+export function fittingString(ctx, str: string, maxWidth: number) {
```

Comment:
```suggestion
export function clipText(ctx, str: string, maxWidth: number) {
```
maybe ? or `clipTextWithEllipsis` which is even clearer ?